.github/workflows/vale-linting.yml

```diff
@@ -1,26 +1,24 @@
 name: Vale Documentation Linting
 
 on:
-  pull_request_target:  # ← This is the key change
-    types: [opened, synchronize, reopened]
+  pull_request:
     paths:
       - '**.md'
       - '**.adoc'
 
 permissions:
   contents: read
-  pull-requests: write
+  pull-requests: write  # Required for posting PR comments
```

Comment:
This will only work as long as it's a PR from an upstream, otherwise it will not work for forked PRs